### PR TITLE
add Random test group

### PR DESCRIPTION
**Pull request by ayrat555**

### CODE REVIEW

File: test/evm_test.exs [hunk 1]
```diff
@@ -168,22 +189,31 @@ defmodule EvmTest do
...
       if Map.get(@passing_tests_by_group, test_group_name) == :all do
         all_tests_of_type(test_group_name)
       else
         Map.get(@passing_tests_by_group, test_group_name)
       end
-      |> Enum.map(fn test_name ->
-        {test_name, read_test_file(test_group_name, test_name)}
-      end)
+
+    Enum.map(tests, fn test_data ->
```
Comment (masonforest) [inline test/evm_test.exs]: do we need the variable at all? ie:

    if Map.get(@passing_tests_by_group, test_group_name) == :all do
     all_tests_of_type(test_group_name)
    else
      Map.get(@passing_tests_by_group, test_group_name)
    end |>
      Enum.map(fn test_data ->
       ...

Comment (ayrat555) [reply test/evm_test.exs]: This version caused credo warning `Pipe chain should start with a raw value`

Comment (ayrat555) [reply test/evm_test.exs]: so it's ok?

Comment (masonforest) [reply test/evm_test.exs]: ah, yep 👍

**Pull request merged**